partition/roles/sonic/templates/frr.conf.j2

```diff
@@ -18,6 +18,14 @@ vrf {{ i.vrf }}
 exit-vrf
 {% endif %}
 {% endfor %}
+{% if sonic_vrfs is defined and sonic_vrfs|length > 0 %}
+{% for vrf in sonic_vrfs %}
+!
+vrf Vrf{{ vrf.id }}
+ vni {{ vrf.vni }}
+exit-vrf
+{% endfor %}
+{% endif %}
```

Comment:
What is the difference to `sonic_interconnects`?

Reply:
This was left over from an earlier configuration experiment. I have deleted it for now; if something like this is necessary later on it can be recreated easily enough.